web-app/src/main/webapp/css/style.css

```diff
@@ -1,9 +1,26 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+#map {
+  /* placeholder until we add a map */
+  background-color: #add8e6;
+  height: 100vh;
+  position: absolute;
+  width: 100%;
+  z-index: 1;
 }
 
-#greeting-container {
-  margin-top: 20px;
+#addressForm {
+  /* overwrite skeleton margins on form */
+  margin-top: 50px;
+  margin-bottom: 0;
+  position: absolute;
+  width: 100%;
+  z-index: 2;
```

Comment:
Is the address form actually an 'overlay' above the map or just a separate page? If it's a separate page, just hide the map via JS.